optuna_dashboard/_serializer.py

```diff
@@ -105,7 +105,7 @@ def serialize_frozen_trial(study_id: int, trial: FrozenTrial) -> Dict[str, Any]:
         "state": trial.state.name.capitalize(),
         "params": [{"name": name, "value": str(value)} for name, value in trial.params.items()],
         "user_attrs": serialize_attrs(trial.user_attrs),
-        "system_attrs": serialize_attrs(trial.system_attrs),
+        "system_attrs": serialize_attrs(getattr(trial, "_system_attrs", {})),
```

Comment:
Use `getattr` here since `trial._system_attrs` is a private method.

Reply:
LGTM!